### PR TITLE
 Adding cosmos db Quickstart template

### PR DESCRIPTION
**Pull request by SnehaGunda**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by SnehaGunda** [comment]
@markjbrown can you take a look at this PR, not sure why the validation failed.

**Comment by mumian** [comment]
@SnehaGunda - check the two href in the readme file, 101-cosmosdb-sql shall be 101-cosmosdb-create

**Comment by SnehaGunda** [comment]
#sign-off

**Comment by SnehaGunda** [comment]
@bmoore-msft Can you please merge this PR.

**Comment by SnehaGunda** [comment]
#sign-off

File: 101-cosmosdb-create/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Create an Azure Cosmos DB account for Core (SQL) API, a database, and a container 
+
```
Comment (bmoore-msft) [inline 101-cosmosdb-create/README.md]: add badges (see ci results)

File: 101-cosmosdb-create/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+		"secondaryRegion":{
+			"type":"string",
+			"metadata": {
+			  "description": "The secondary replica region for the Cosmos DB account."
+		  }
+		},
+		"defaultConsistencyLevel": {
+			"type": "string",
+			"defaultValue": "Session",
+			"allowedValues": [ "Eventual", "ConsistentPrefix", "Session", "BoundedStaleness", "Strong" ],
```
Comment (bmoore-msft) [inline 101-cosmosdb-create/azuredeploy.json]: format the json, e.g. SHIFT+ALT+F in VS Code

File: 101-cosmosdb-create/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+					"indexingPolicy": {
+						"indexingMode": "consistent",
+						"includedPaths": [{
+								"path": "/*"
+							}
+						],
+						"excludedPaths": [{
+								"path": "/myPathToNotIndex/*"
+							}
+						],
```
Comment (bmoore-msft) [inline 101-cosmosdb-create/azuredeploy.json]: remove extraneous comma

File: 101-cosmosdb-create/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+		{
+			"type": "Microsoft.DocumentDB/databaseAccounts/sqlDatabases",
+			"name": "[concat(variables('accountName'), '/', parameters('databaseName'))]",
+			"apiVersion": "2019-08-01",
+			"dependsOn": [ "[resourceId('Microsoft.DocumentDB/databaseAccounts', variables('accountName'))]" ],
+			"properties":{
+				"resource":{
+					"id": "[parameters('databaseName')]"
+				},
+				"options": {}
```
Comment (bmoore-msft) [inline 101-cosmosdb-create/azuredeploy.json]: remove empty property

File: 101-cosmosdb-create/README.md (hunk 1 above)
Comment (SnehaGunda) [reply 101-cosmosdb-create/README.md]: Aren't the labels/badges automatically added?

**Comment by SnehaGunda** [comment]
#sign-off

**Pull request merged**